packages/good-design/src/apps/ubi/screens/StartClaim.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, useEffect } from "react";
+import { Center, VStack } from "native-base";
+import { useWizard } from "react-use-wizard";
+
+import { TransTitle, TransText } from "../../../core/layout";
+import { Web3ActionButton } from "../../../advanced";
+import { useClaimContext } from "../context";
+
+export const StartClaim: FC = () => {
+  const { account, chainId, supportedChains, onConnect } = useClaimContext();
+  const { nextStep } = useWizard();
+
+  useEffect(() => {
```

Comment:
whats the purpose of this screen?
why does it trigger nextstep automatically?

Reply:
There is screen shown for non-connected wallet. (gooddapp)

It triggers the next-step to go the pre-claim screen if a user connects or is already connected.

reference: https://www.figma.com/design/ihw1PxBvLxacTHnN2aj4lC/3.-Product?node-id=19551-115502&t=6IHmcgnQjWVqrR6E-4